mkposters/mkposter.py

```diff
@@ -53,17 +53,13 @@ def mkposter(datadir):
         {banner}
         </div>
         <hr>
-        <div class="body md-typeset">
-        <div class="left">
-        {left_body}
-        </div>
-        <div class="right">
-        {right_body}
-        </div>
+        <div class="body md-typeset">""",
+        "\n".join([f"<div class=\"Coll{i}\">\n{body}\n</div>" for i, body in enumerate(bodies[1:])]),
```

Comment:
Nit: we can probably skip the nested `join`, by writing this line as `*[f"..." for ... in ....],`

(so that the earlier join handles everything)